src/applications/mhv/medications/components/PrescriptionDetails/NonVaPrescription.jsx

```diff
@@ -88,7 +88,12 @@ const NonVaPrescription = prescription => {
           <h3 className="vads-u-font-size--base vads-u-font-family--sans">
             Provider notes
           </h3>
-          <p>{validateField(prescription.remarks)}</p>
+          <p>
+            {validateField(
+              (prescription.remarks ?? '') +
+                (prescription.disclaimer ? ` ${prescription.disclaimer}` : ''),
```

Comment:
is it possible that remarks is null or undefined? If so, would the disclaimer string start with a space?